tests/v2/actions/secrets/template.go

```diff
@@ -0,0 +1,18 @@
+package secrets
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewSecretTemplate is a constructor that creates the secret template for secrets
+func NewSecretTemplate(secretName, namespaceName string, data map[string][]byte, secType corev1.SecretType) corev1.Secret {
```

Comment:
similar to configmaps comment, should this be considered an extension as its more of a template or a constructor where we are creating and returning secret object..  we are not performing any crud operations and no config is involved here?